driver-sync/src/test/functional/com/mongodb/client/JsonPoweredCrudTestHelper.java

```diff
@@ -305,11 +305,10 @@ BsonDocument getRunCommandResult(final BsonDocument collectionOptions, final Bso
                 response = database.runCommand(clientSession, command, readPreference, BsonDocument.class);
             }
         }
-        response.remove("ok");
```

Comment:
According to the [legacy test format specification](https://github.com/mongodb/specifications/blob/master/source/transactions/tests/legacy-test-format.md#use-as-integration-tests): 

> If the operation returns a raw command response, e.g., from runCommand, then compare only the fields present in the expected result document.

Therefore, we should not remove any fields from the response. Instead, we should only compare the fields that are explicitly expected and ignore any additional fields that are not specified in the expected results.